jolt-core/src/subprotocols/twist.rs

```diff
@@ -374,7 +374,16 @@ fn prove_read_write_checking_local<F: JoltField, ProofTranscript: Transcript>(
     );
     let _guard = span.enter();
 
-    // Data structure described in Equation (72)
+    // The table I is used to update the Val table efficiently when we compute the univariate polynomial
+    // that the prover sends to the verifier in each round. Specifically, I stores data of
+    // the form (j, k, INC * LT, INC), where j indexes the instruction cycle, k is the
+    // the memory location written, and INC * LT represents the sum of elements
+    // Inc(k, j, j') * Lt(j', r) as j' ranges over {0, 1}^(log(chunk_size) - i).
+    // for round i, where r = (r_1, ..., r_i) is the vector of committed cycle variables in F.
+    // Similarly, INC represents the sum of Inc(k, j, j') over the same range of j'.
+    // Table I in round i has size T / 2^i.
```

Comment:
The number of (nonzero) elements in I shrinks by ≤2 in each round, depending on locality. 
If you view I as a matrix, however, the number of *rows* in round i is T / 2^i